pkg/operator/controller/controller.go

```diff
@@ -182,6 +194,24 @@ func (r *reconciler) enforceEffectiveIngressDomain(ci *operatorv1.IngressControl
 	return nil
 }
 
+// ensureUniqueDomain ensures spec.domain from ic is unique across all ingress controllers
+// in the same namespace.
+func (r *reconciler) ensureUniqueDomain(ic *operatorv1.IngressController) (bool, error) {
```

Comment:
Function name doesn't seem to quite match the behavior — given the signature I'd have expected something like `isDomainUnique` or something

Reply:
Good point. I'll update.